assets/js/state/sagas/softwareUpdates.test.js

```diff
@@ -0,0 +1,88 @@
+import { faker } from '@faker-js/faker';
+
+import { recordSaga } from '@lib/test-utils';
+
+import { networkClient } from '@lib/network';
+import MockAdapter from 'axios-mock-adapter';
+
+import {
+  startLoadingSoftwareUpdates,
+  setSoftwareUpdates,
+  setSoftwareUpdatesErrors,
+  setEmptySoftwareUpdates,
+} from '@state/softwareUpdates';
+
+import { fetchSoftwareUpdates } from './softwareUpdates';
+
+describe('Software Updates saga', () => {
+  describe('Fetching Software Updates', () => {
+    it('should successfully fetch software updates', async () => {
+      const axiosMock = new MockAdapter(networkClient);
+      const hostId = faker.string.uuid();
+      const successfulResponse = [
+        {
+          date: '2023-05-18',
+          advisory_name: 'SUSE-15-SP4-2023-2245',
+          advisory_type: 'Bug Fix Advisory',
```

Comment:
If I recall it correctly, we do a little transformation in the backend for this field only.
Advisory type would be one of:
- `security_advisory`
- `bugfix`
- `enhancement`

but needs to be double checked against the actual api response, when ready.